src/azure-cli-core/azure/cli/core/adal_authentication.py

```diff
@@ -171,3 +171,13 @@ def _try_scopes_to_resource(scopes):
 
     # Exactly only one scope is provided
     return scopes_to_resource(scopes)
+
+
+class BasicTokenCredential:
+    # pylint:disable=too-few-public-methods
+    """A Track 2 implementation of msrest.authentication.BasicTokenAuthentication."""
+    def __init__(self, access_token):
+        self.access_token = access_token
+
+    def get_token(self, *scopes, **kwargs):  # pylint:disable=unused-argument
+        return AccessToken(self.access_token, int(time.time() + 3600))
```

Comment:
Can we use a meaningful constant instead of a magic number?

Reply:
As `self.access_token` is a fixed value, `get_token` cannot refresh the token, thus defeating the purpose of returning a valid `expires_on`. Using `time.time() + 3600` will force SDK to use the provided token, regardless of whether the token is expired or not.

Also, [Subscriptions - List](https://docs.microsoft.com/en-us/rest/api/resources/subscriptions/list) and [Tenants - List](https://docs.microsoft.com/en-us/rest/api/resources/tenants/list) APIs are not long-running operations, so the time of `expires_on` will not be reached anyway.